config/puma.rb

```diff
@@ -0,0 +1,56 @@
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
I generally prefer puma, but the default in this app is using unicorn. The rails upgrader created this file, but it should probably be removed?